databricks/koalas/frame.py

```diff
@@ -1963,6 +1963,68 @@ def sort_values(self, by: Union[str, List[str]], ascending: Union[bool, List[boo
         else:
             return kdf
 
+    def sort_index(self, axis: int = 0, level: int = None, ascending: bool = True,
+                   inplace: bool = False, kind: str = None, na_position: str = 'last') \
+            -> Optional['DataFrame']:
+        """
+        Sort object by labels (along an axis)
+
+        Parameters
+        ----------
+        axis : index, columns to direct sorting
+        level : int or level name or list of ints or list of level names
+            if not None, sort on values in specified index level(s)
+        ascending : boolean, default True
+            Sort ascending vs. descending
+        inplace : bool, default False
+            if True, perform operation in-place
+        kind : Koalas does not allow specifying the sorting algorithm at the moment, default None
```

Comment:
`Koalas does not allow specifying the sorting algorithm at the moment` should be in the next line.

Reply:
Fair point. I've fixed it with c899fde.